include/natalie/random_value.hpp

```diff
@@ -21,6 +21,8 @@ class RandomValue : public Value {
     RandomValue()
         : Value { Value::Type::Random, GlobalEnv::the()->Random() } { }
 
+    ~RandomValue() { free(m_generator); }
```

Comment:
Let's use `delete m_generator` here to match the use of `new` in the initializer.